android/src/main/java/com/reactnativecommunity/webview/RNCWebViewMapManager.kt

```diff
@@ -0,0 +1,5 @@
+package com.reactnativecommunity.webview
+
+object RNCWebViewMapManager {
+  val rncWebViewMap = mutableMapOf<String, RNCWebView>()
+}
```

Comment:
nit: We can add a newline here to make that red github symbol go away

Reply:
done